storops_test/lib/test_converter.py

```diff
@@ -226,3 +226,19 @@ def test_url_to_mask_not_found(self):
         url = 'https://10.244.211.30/router'
         ret = converter.url_to_mask(url)
         assert_that(ret, none())
+
+    def test_to_minute(self):
+        r = converter.to_minute("00:06:00.000")
+        assert_that(r, equal_to(6))
+
+    def test_to_hour(self):
+        r = converter.to_hour("07:00:00.000")
+        assert_that(r, equal_to(7))
+
+    def test_from_minute_str(self):
```

Comment:
What if minute is larger than 60?  Please add a test case to demonstrate the output.